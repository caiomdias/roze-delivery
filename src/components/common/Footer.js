import React from 'react';

const Footer = () => (
	<footer className="footer">
		<div className="container">
			<span>Â© 2020 Roze Delivery</span>
		</div>
	</footer>
);

export default Footer;
