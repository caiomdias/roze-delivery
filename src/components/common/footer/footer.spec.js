import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe("Footer Component", () => {
	it('render correctly Footer component', () => {  
		const footerComponent = renderer.create(<Footer />);
		expect(footerComponent).toMatchSnapshot();
	});
});