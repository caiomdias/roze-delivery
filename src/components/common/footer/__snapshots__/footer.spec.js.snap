// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component render correctly Footer component 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <span>
      Â© 2020 Roze Delivery
    </span>
  </div>
</footer>
`;
